lib/pgrel/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module Pgrel # :nodoc:
-  VERSION = "0.1.3"
+  VERSION = "0.2.0"
```

Comment:
Generally, it's not a good idea to bump versions in PRs. But let's leave it this time